changelog/README.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 25th October 2024

```

Comment:
issue: Please don't forget to update the date once the change is deployed to production (the changelog tracks the date when the change was made to docs, not necessarily when it got released).

Reply:
@jnv I can see the change has been deployed to production today as the Swagger docs have updated and reflect the changes I made - https://app.mews.com/swagger/index.html?urls.primaryName=connector#/Configuration/configuration_get

I've updated the changelog for today 🙂